packages/button-theme/__tests__/DatePicker.test.tsx

```diff
@@ -8,6 +8,11 @@ import 'regenerator-runtime/runtime';
 expect.extend(toHaveNoViolations);
 
 describe('DatePicker', () => {
+  beforeAll(() => {
+    // TODO: Remove this once https://github.com/nickcolley/jest-axe/issues/147 is fixed.
+    window.getComputedStyle = ():any => {};
+  });
+
```

Comment:
You can read more about this issue [here](https://github.com/NickColley/jest-axe/issues/147) and [here](https://github.com/jsdom/jsdom/issues/3025). Without this code, the tests still pass however they have noisy output like 
```
PASS __tests__/Textarea.test.tsx
@button-inc/button-theme:   ● Console
@button-inc/button-theme:     console.error
@button-inc/button-theme:       Error: Not implemented: window.computedStyle(elt, pseudoElt)
@button-inc/button-theme:           at module.exports (/Users/mastro/button/service-development-toolkit/node_modules/jsdom/lib/jsdom/browser/not-implemented.js:9:17)
@button-inc/button-theme:           at Window.getComputedStyle (/Users/mastro/button/service-development-toolkit/node_modules/jsdom/lib/jsdom/browser/Window.js:795:7)
@button-inc/button-theme:           at hasPsuedoElement2 (/Users/mastro/button/service-development-toolkit/node_modules/axe-core/axe.js:16285:26)
@button-inc/button-theme:           at /Users/mastro/button/service-development-toolkit/node_modules/axe-core/axe.js:2253:30
@button-inc/button-theme:           at Object.colorContrastEvaluate (/Users/mastro/button/service-development-toolkit/node_modules/axe-core/axe.js:16331:13)
@button-inc/button-theme:           at Check.Object.<anonymous>.Check.run (/Users/mastro/button/service-development-toolkit/node_modules/axe-core/axe.js:18562:34)
@button-inc/button-theme:           at /Users/mastro/button/service-development-toolkit/node_modules/axe-core/axe.js:18698:18
@button-inc/button-theme:           at pop (/Users/mastro/button/service-development-toolkit/node_modules/axe-core/axe.js:6198:18)
@button-inc/button-theme:           at Object.defer (/Users/mastro/button/service-development-toolkit/node_modules/axe-core/axe.js:6220:11)
@button-inc/button-theme:           at /Users/mastro/button/service-development-toolkit/node_modules/axe-core/axe.js:18697:20 undefined
@button-inc/button-theme:       at VirtualConsole.<anonymous> (../../node_modules/jsdom/lib/jsdom/virtual-console.js:29:45)
@button-inc/button-theme:       at module.exports (../../node_modules/jsdom/lib/jsdom/browser/not-implemented.js:12:26)
@button-inc/button-theme:       at Window.getComputedStyle (../../node_modules/jsdom/lib/jsdom/browser/Window.js:795:7)
@button-inc/button-theme:       at hasPsuedoElement2 (../../node_modules/axe-core/axe.js:16285:26)
@button-inc/button-theme:       at ../../node_modules/axe-core/axe.js:2253:30
@button-inc/button-theme:       at Object.colorContrastEvaluate (../../node_modules/axe-core/axe.js:16331:13)
@button-inc/button-theme:       at Check.Object.<anonymous>.Check.run (../../node_modules/axe-core/axe.js:18562:34)
```

Reply:
It looks like that workaround [might break some functionality](https://github.com/NickColley/jest-axe/issues/147#:~:text=This%20workaround%20appears%20to%20break%20functionality%20in%20jest%2Daxe). Relevant for us?

Reply:
Good call. Since the broken functionality is not well understood, I will remove the workaround code and create a tech-debt card for this.